src/components/item/form/link/LinkForm.tsx

```diff
@@ -0,0 +1,393 @@
+          endAdornment: (
+            <>
+              {isDescriptionDirty ? (
+                <IconButton
+                  onClick={onClickRestoreDefaultDescription}
+                  sx={{
+                    visibility:
+                      description !== linkData?.description
+                        ? 'visible'
+                        : 'hidden',
+                  }}
+                >
+                  <Undo2Icon size="20" />
+                </IconButton>
+              ) : undefined}
+              <IconButton
+                onClick={onClickClearDescription}
+                sx={{
+                  visibility: description ? 'visible' : 'hidden',
+                }}
+              >
+                <ClearIcon fontSize="small" />
+              </IconButton>
+            </>
+          ),
+        }}
+      />
+      <FormControl>
+        <FormLabel>
+          <Typography variant="caption">
+            {translateBuilder(BUILDER.CREATE_ITEM_LINK_TYPE_TITLE)}
+          </Typography>
+          {linkContent ? (
+            <RadioGroup
+              name="use-radio-group"
+              value={linkType}
+              onChange={onChangeLinkType}
+              sx={{ display: 'flex', gap: 2 }}
+            >
+              <StyledFormControlLabel
+                value={LinkType.Default}
+                label={<Link href={url}>{linkContent}</Link>}
+                control={<Radio />}
+              />
+              <StyledFormControlLabel
+                value={LinkType.Fancy}
+                label={
+                  <LinkCard
+                    title={linkData?.title || ''}
+                    url={linkContent}
+                    thumbnail={linkData?.thumbnails[0]}
+                    description={description || ''}
+                  />
+                }
+                control={<Radio />}
+                slotProps={{ typography: { width: '100%', minWidth: '0px' } }}
+                sx={{ minWidth: '0px', width: '100%' }}
+              />
+              {linkData?.isEmbeddingAllowed && (
+                <StyledFormControlLabel
+                  value={LinkType.Embedded}
+                  label={embeddedLinkPreview}
+                  control={<Radio />}
+                  slotProps={{
+                    typography: {
+                      width: '100%',
+                      minWidth: '0px',
+                    },
+                  }}
+                  sx={{
+                    // this ensure the iframe takes up all horizontal space
+                    '& iframe': {
+                      width: '100%',
+                    },
+                  }}
+                />
+              )}
+              {linkData?.html && linkData.html !== '' && (
+                <StyledFormControlLabel
+                  value={LinkType.Embedded}
+                  label={
+                    // eslint-disable-next-line react/no-danger
+                    <StyledDiv
+                      sx={{}}
```

Comment:
it is necessary to have an empty sx ? If yes, maybe we should make it optional in the StyledDiv.